components/Card/Card.tsx

```diff
@@ -0,0 +1,179 @@
+"use client";
+
+import React from "react";
+import { useRef, useState } from "react";
+
+import Image from "next/image";
+
+import AddModal from "@/components/AddModal/AddModal";
+import DeleteLinkModal from "components/DeleteLinkModal/DeleteLinkModal";
+import useOutsideClick from "hooks/useOutsideClick";
+import beautifyDate from "lib/beautifyData";
+import { ILink } from "lib/getFolderData";
+
+import styles from "./Card.module.css";
+
+interface ICard {
+  link: ILink;
+  idx: number;
+  isClicked: boolean;
+  setIsClickedKebab: React.Dispatch<React.SetStateAction<number>>;
```

Comment:
특별한 목적이 아니라면 이렇게 작성하는건 어떨까요?
```suggestion
  setIsClickedKebab: (isClickedKebab: number) => void;
```

Reply:
setIsClickedKebab의 경우 boolean을 파라미터로 받을것 같지만 실제로는 number를 받는데 idx를 받을거라면 setIdx, setClickedIdx 가 어떨까요? 
+ 개인적으로 idx보단 index가 더 명시적인것 같습니다.